### PR TITLE
Adblock message tablet fix

### PR DESCRIPTION
**Pull request by Calanthe**
Before:
screen shot 2015-07-22 at 11 59 06

After:
screen shot 2015-07-22 at 11 59 16

cc @sndrs @davidrapson

### CODE REVIEW

File: static/src/javascripts/projects/common/views/donot-use-adblock.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
     <%=messageText%></p>
 <ul class="site-message__actions u-unstyled">
```
Comment (uplne) [inline static/src/javascripts/projects/common/views/donot-use-adblock.html]: What is the difference between this template and `membership-message.html` template. Can we reuse it?

Comment (Calanthe) [inline static/src/javascripts/projects/common/views/donot-use-adblock.html]: You are right. I will use the same template.

**Comment by Calanthe** [comment]
Now also the arrow icon is fixed. I moved the whole 'adblock-message' code to dedicated module and reused the membership message template.

**Comment by sndrs** [comment]
lgtm :+1:

**Comment by davidrapson** [comment]
:+1: LGTM too

File: static/src/javascripts/bootstraps/common.js [hunk 2]
```diff
@@ -49,7 +50,6 @@ define([
     'common/modules/save-for-later',
     'text!common/views/international-message.html',
     'text!common/views/international-control-message.html',
-    'text!common/views/donot-use-adblock.html',
```
Comment (sndrs) [inline static/src/javascripts/bootstraps/common.js]: the corresponding module name was not removed from module names list

**Pull request merged**